travis.cfg

```diff
@@ -1,35 +1,44 @@
 [buildout]
-extends = 
+extends = development.cfg
```

Comment:
aqui, @idgserpro

Reply:
Sim, por isso no .travis.yml tem de fazer agora `test $MASTER && sed -ie '/buildout\.d\/versions\.cfg/d' buildout.d/development.cfg || true` ou o `versions.cfg` do portal.buildout estará presente quando for testar o `MASTER` o que não é correto [(pois ele está no extends do `development.cfg`)](https://github.com/plonegovbr/portal.buildout/blob/ccc8780ed863900e253ad94710c0641f67f8bc40/development.cfg#L5).

Reply:
não ficou claro; pode fazer essa mudança para mim, por favor?

Reply:
Feito em https://github.com/plonegovbr/portal.buildout/commit/7379a53072b8a36fab42d8451425061385745240

Reply:
agora entendi.